pkg/partmgr/partmgr.go

```diff
@@ -170,15 +171,32 @@ func (p *PartitionManager) createAndUpdatePartition(t int64) (*DBPartition, erro
 	return partition, err
 }
 
```

Comment:
you can create a metrics reporter instance from config at v3iotsdb.go:133 add pass it to partmgr constructor thus avoiding calling DefaultReporterInstance each time and also using the right config